src/components/AppBanner.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+  export let message = "";
+  export let bgColor;
+</script>
+
+<div
+  style="background-color: {bgColor}"
+  class="alert-banner flex items-center justify-between w-full p-2 shadow text-white">
```

Comment:
What is the purpose of the `alert-banner` class? I can't seem to find it on [tailwind's documentation](https://tailwindcss.com/components/alerts#banner). I see that they add a `role="alert"` to alerts, let's add that too.